src/javascript/Create/CreateForm/create.request.js

```diff
@@ -18,7 +18,7 @@ export const createNode = ({
         language
     }
 }) => {
-    const {propsToSave, mixinsToAdd} = getDataToMutate({}, values, sections, language);
+    const {propsToSave, mixinsToAdd} = getDataToMutate(nodeData, values, sections, language);
```

Comment:
This is not the way to fix this issue, `nodeData` in create action is the parent node data and is used by `getDataToMutate` to get the  properties to modify. 
We need to handle it in a different way.